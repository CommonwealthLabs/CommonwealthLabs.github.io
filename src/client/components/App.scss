body {
  margin: 0;
  background: #F4F6F8;
}

h1, h2, h3, h4, h5, h6 {
  margin: 0;
}

ul {
  margin: 0;
  padding: 0;
}

.app-style {
  font-size: 3rem;
}

//
// .app-style
//   font-size: 3rem
// }
