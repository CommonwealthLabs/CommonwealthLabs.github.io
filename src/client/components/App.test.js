// import test from 'ava';

import React from 'react';
// import register from 'ignore-styles'
// register(['.sass', '.scss'])
import { App } from './App.react';
// import sum from './sum';

import renderer from 'react-test-renderer';

test('App renders concatenated string', () => {
  const app = renderer.create(<App a="Testing " b="this!" />).toJSON();
  expect(app).toMatchSnapshot();
});
