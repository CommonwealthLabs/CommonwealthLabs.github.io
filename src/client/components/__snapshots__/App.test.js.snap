// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders concatenated string 1`] = `
<div
  styleName="app-style"
>
  Testing this!
</div>
`;
